tvaultuiv2/src/views/private/certificates/components/CertificatesDashboard/index.js

```diff
@@ -642,7 +644,7 @@ const CertificatesDashboard = () => {
                 onDeleteCertificateClicked(certificate)
               }
               onEditListItemClicked={() => onEditListItemClicked(certificate)}
-              admin={contextObj.isAdmin}
+              admin
```

Comment:
should it be hardcoded?
```suggestion
              admin
```